controllers/question.controller.js

```diff
@@ -0,0 +1,42 @@
+const utility = require('../lib/utility.lib');
+const { createQuestion, destroyQuestion } = require('../lib/question.lib');
+
+const {
+  filterValues, formatValues,
+} = utility;
+
+const postQuestion = async (req, res) => {
+  try {
+    const rawData = req.body;
+    const filteredValues = filterValues(rawData, ['question_text']);
+    const data = formatValues(filteredValues);
+
+    const question = await createQuestion({
+      question_text: data.question_text,
+      user_id: 1,
+    });
+
+    if (question) {
+      return res.status(200).send({
```

Comment:
I think that check is a bit unnecessary because `createQuestion` will not return `undefined` or `null`

Reply:
So what happens if there is an error?

Reply:
If there is an error, the error will be handled by the try-catch in lib file, the error message is re-thrown in the lib file which is handled again in the try-catch in the controller file where 500 status is returned to the API consumer. So `question` will always have a value except you modify the lib later such that there is a tendency for it to return null or undefined.